resource/localization/de/npc_spawn_platforms.properties

```diff
@@ -0,0 +1,79 @@
+tool.npc_spawnplatform.oldspawning=Alten Spawn-Modus verwenden
+tool.npc_spawnplatform.oldspawning.help=Standardm\u00E4\u00DFig h\u00E4lt die Spawn-Uhr an, wenn die Plattform voll ist. Mit dieser Option l\u00E4uft die Uhr weiter.
+tool.npc_spawnplatform.killvalue=Belohnung f\u00FCr T\u00F6ten
+tool.npc_spawnplatform.killvalue.help=Manche Spielmodi und Addons verteilen Belohnungen f\u00FCr get\u00F6tete NPCs. Ein anderer Wert als -1 \u00FCberschreibt diese Belohnung.
+
+tool.npc_spawnplatform.weapon_rebel=Zuf\u00E4llige Rebel-Waffe
+tool.npc_spawnplatform.weapon_combine=Zuf\u00E4llige Combine-Waffe
+tool.npc_spawnplatform.weapon_citizen=Zuf\u00E4llige Citizen-Waffe
+
+tool.npc_spawnplatform.overlay=NPC: %s\u000AWaffe: %s\u000AVerz\u00F6gerung: %s\u000AMaximum lebendig: %d
+tool.npc_spawnplatform.overlay_flipped=Kann %ss in diesem Winkel nicht spawnen!
+
+
+# Sandbox
+Cleanup_Spawnplatforms=NPC-Spawn-Plattformen
+Cleaned_Spawnplatforms=Alle NPC-Spawn-Plattformen aufger\u00E4umt
+
+# Settings menu
+spawnmenu.utilities.spawnplatform=NPC-Spawn-Plattformen
+utilities.spawnplatform.cleanupcorpses=Leichen aufr\u00E4umen
+utilities.spawnplatform.cleanupcorpses.help=Alle Leichen jede Minute automatisch aufr\u00E4umen
+utilities.spawnplatform.adminonly=Nur Admins
+utilities.spawnplatform.adminonly.help=Verhindert dass normale User Spawn-Plattform erstellen
+utilities.spawnplatform.callhooks=Sandbox-Hooks verwenden
+utilities.spawnplatform.callhooks.help=Die Plattform benimmt sich als h\u00E4tte der User das Spawn-Men\u00FC verwendet um NPCs zu erstellen, sodass Anzahllimits u.\u00E4. eingehalten werden.
+utilities.spawnplatform.maxinplay=Maximale NPCs pro Plattform
+utilities.spawnplatform.maxinplay.help=Wie viele lebendige NPCs eine einzelne Plattform haben kann
+utilities.spawnplatform.mindelay=Minimale Spawn-Verz\u00F6gerung
+utilities.spawnplatform.mindelay.help=Die Mindestzeit, die eine Plattform warten muss, bevor ein neuer NPC gespawnt werden kann
+utilities.spawnplatform.debug=Entwickleroutput aktivieren
+utilities.spawnplatform.debug.help=Diagnostiknachrichten aktivieren oder deaktivieren. Die convar 'developer' muss 1 oder 2 sein.
+
+utilities.spawnplatform.dangerzone=Gefahrenzone
+utilities.spawnplatform.sanity=NPC-Liste erzwingen
+utilities.spawnplatform.sanity.help=Nur NPCs auf der NPC-Liste spawnen. Wenn diese Option deaktiviert ist, k\u00F6nnen Spieler alle Einheiten spawnen und eventuell Serverabst\u00FCrze oder Schlimmeres verursachen.
+utilities.spawnplatform.rehydrate=Missing NPCs on Dupe Fix
+utilities.spawnplatform.rehydrate.help=Eine Plattform, die ohne ihre NPCs kopiert wurde, spawnt alle NPCs der alten Plattform new.\u000ADies ist wichtig f\u00FCr Speicherdateien und Persistence (welche der Duplicator intern verwendet), aber kann beim Benutzen des Duplicators zu unerwarteten Ergebnisse f\u00FChren.\u000ABenutzer, die Persistence Mode nicht verwenden, k\u00F6nnen diese Einstellung deaktivieren.
```

Comment:
Missed this and need to rewrite the helptext after today's conversation. Will fix.